how-tos/access_data.qmd

```diff
@@ -2,11 +2,6 @@
 title: "How do I access data?"
 ---
 
-Here we'll cover: 
-
-- [How do I access data If I'm in the cloud?](if-im-in-the-cloud.qmd)
-- [How do I access data if I'm on my local machine?](if-im-local.qmd)
```

Comment:
To revisit and update depending on the "No TOC on landing page" decision.

Reply:
restored that text (in my local, to be pushed)